src/main/resources/alma/fix/subjects.fix

```diff
@@ -542,3 +542,14 @@ do list(path:"spatial[]", "var":"$i")
   replace_all("$i.focus.geo.lon","^Point\\((.*) .*\\)","$1")
   replace_all("$i.focus.geo.lat","^Point\\(.* (.*)\\)","$1")
 end
+
+set_array("digiBibSubjectFacet[]")
+do list(path:"subject[]","var":"$i")
+  if any_equal("$i.type","Concept")
```

Comment:
Shouldn't this be `$i.type[]`?

Looking at test case `990197293880206441.json`, it should also have "Computer science", "Software engineering" and "Operating systems (Computers)" in the new field.

Reply:
You are right